.github/workflows/deploy_edge.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy Edge
+
+on:
+  workflow_run:
+    workflows: ["Node.js CI"]
+    types:
+      - completed
+jobs:
+  deply-edge-package:
+    runs-on: self-hosted
+    if: ${{ github.event_name == 'pull_request' || github.ref == 'refs/heads/master' }}
+    strategy:
+      matrix:
+        node: [12.x]
+    env:
+      VERDACCIO_TOKEN: ${{ secrets.EDGE_VERDACCIO_TOKEN }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - run: echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
+    - run: echo "PACKAGE_VERSION=`node -p "require('./packages/taquito/package.json').version"`" >> $GITHUB_ENV
+    - run: echo "BRANCH_NAME=`echo $GITHUB_REF | cut -d'/' -f 3`" >> $GITHUB_ENV
+    - run: echo "TARGET_VERSION=${PACKAGE_VERSION}-${SHORT_SHA}--${BRANCH_NAME}" >> $GITHUB_ENV
+    - run: npm i -g lerna
```

Comment:
I don't think you need to install lerna globally, as it gets installed by `npm ci` and when you run `npm run lerna...` npm finds the local copy of lerna under `node_modules/`